lib/mariaex/connection.ex

```diff
@@ -208,7 +208,9 @@ defmodule Mariaex.Connection do
   end
 
   defp command({:query, statement, params, _opts}, s) do
-    Protocol.send_query(statement, params, s)
+    statement
```

Comment:
It would not be the right place to do it. Traversing of the whole string can have performance impact.